### DISCUSSION TITLE
Unable to  set front-page title for book-mode PDF

### DISCUSSION THREAD

**Discussion by johnkerl** [quarto-dev/quarto-cli#3988] [Q&A] [answered]
Hello -- I've been through several doc pages including https://quarto.org/docs/reference/projects/books.html and I haven't found the answer to the following. My apologies if this is an RTM!

* Repro repo: https://github.com/johnkerl/quarto-book-mode
* I attempt to set the title various ways:
  * https://github.com/johnkerl/quarto-book-mode/blob/main/_quarto.yml#L5
  * https://github.com/johnkerl/quarto-book-mode/blob/main/index.qmd#L2
  * https://github.com/johnkerl/quarto-book-mode/blob/main/index.qmd#L4
* With `quarto preview .` (HTML) I get the first image
  * One of the three above choices appears as the title
* With `open _book/*.pdf` I get the second image
  * None of my three attempts appears as the first-page title
  * The first-page title is the last-encountered chapter title
 
What's the right way to get the front-page title set for PDF output?

<img width="1119" alt="Screenshot 2023-01-16 at 5 04 38 PM" src="https://user-images.githubusercontent.com/2008794/212772471-7b28684a-efba-4539-b65b-74112332bff0.png">

<img width="850" alt="Screenshot 2023-01-16 at 5 05 24 PM" src="https://user-images.githubusercontent.com/2008794/212772483-3597d2cf-1227-4799-aa51-52a7dc5bdaa7.png">

**Comment by cderv** [comment, marked as answer]
Which version of Quarto are you using ? 

I tried with v1.3.107 pre-release, and can't reproduce. I cloned your project, and ran `quarto render` 
I correctly got the title in `book` filed inside `_quarto.yml` to show as book title.
![image](https://user-images.githubusercontent.com/6791940/212880032-309bd907-598e-42ed-b94f-ca71a46c3900.png)

**Reply by johnkerl** [reply]
@cderv thanks -- my version is indeed outdated! :)
